src/react/components/pages/modelCompose/modelCompose.scss

```diff
@@ -165,7 +166,8 @@ h4 {
 }
 
 .modal-buttons-container {
-    justify-content: space-between;
+    justify-content: flex-end;
+    display: flex;
```

Comment:
I don't think we need 'display: flex' here

Reply:
yes, we do

Reply:
oh, yes you're right